server/app/services/geo/AddressCandidate.java

```diff
@@ -0,0 +1,52 @@
+package services.geo;
+
+import com.google.auto.value.AutoValue;
+
+/** This class is for setting and getting an address candidate */
```

Comment:
suggestion (blocking): remove "this class" and "this method" portions of javadoc in this file as per [style guide](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment)